### DISCUSSION TITLE
Make HR ranked in mania

### DISCUSSION THREAD

**Discussion by swetdep** [ppy/osu#19990] [Ideas]
Although there's not much explanation to give for my part, I don't find a reason why HR is unranked, and it's really frustrating for the players to be unable to get PP out of this mod, and for my part, I have unknowingly lost many good plays because of this.

**Comment by mk56-spn** [comment]
This repo is for Lazer. And stable is feature locked regardless

**Reply by swetdep** [reply]
This is something that has to be talked about and I found this to be the most appropiate place to bring this up.
And, although stable is feature-locked, this problem has been up for more or less than a year.

**Reply by WindowsMeosu** [reply]
I don't know about that one scrubs, If HR were to be ranked I'd imagine that a 1* convert could be like 120pp with DTHR. I mean you could just make it be like HD/FI/FL and not award any pp at all. if HR were to be ranked, peppy would probably make sure it isn't that much overweighted. Of course there is always going to maps where /w HR will be overweighted when it's not even a 4star but that's probably always gonna be a problem.. (also I rushed through this while writing this so I hope this isn't a problem to read or anything) anyway like you and the person above mentioned, stable is feature-locked, and I don't see any good reason to rank HR other than farming pp.

**Reply by swetdep** [reply]
Just like you said; underweight PP if HR is re-ranked, but as I mentioned earlier the situation is really frustrating because of two things:
-You might set good plays in ranked maps with it and might have gotten better score with it than without it. It's a lost play.
-Not letting people submit the beatmap only in mania is unreasonable, as it takes freedom to modify the map and be competitive with this freedom is even more frustrating; How come we can set scores with hr in **all modes** except mania? 
As for the why, it's the same reason why HR exists and is ranked in other modes, besides PP farming; To allow the player to experiment new things, be different, practice or even have more fun without stopping being competitive. We want to have more freedom.

Please, just rank it :c

**Reply by Doksha** [reply]
again this is the lazer page, go to the stable page if you really want this. and as they already said, stable is feature locked, if you want to make suggestions for lazer you can put them here, but not for stable

**Comment by MizuIro-H** [comment]
I was told that there is an algorithm for HR in mania, but it's not released because it wasn't ready and it's suspended now.
Idk exactly how to push them to make HR pp-able, but "simply" multiply the max pp of NM sounds not bad.
(Sorry for poor English)

**Comment by MizuIro-H** [comment]
I was told that there is an algorithm for HR in mania, but it's not released because it wasn't ready and it's suspended now.
Idk exactly how to push them to make HR pp-able, but "simply" multiply the max pp of NM sounds not bad.
(Sorry for poor English)
